.travis.yml

```diff
@@ -168,19 +187,25 @@ jobs:
         - make ${MAKEOPTS} -C ports/unix PYTHON=python2 deplibs
         - make ${MAKEOPTS} -C ports/unix PYTHON=python2 VARIANT=nanbox
         - make ${MAKEOPTS} -C ports/unix PYTHON=python2 VARIANT=nanbox test_full
```

Comment:
If the test fails the following commands will still execute and the test failure files will be deleted/overwritten.  I think this can be fixed by adding to this line: ` || travis_terminate 1`